tests/core/editable/manual/focus.html

```diff
@@ -0,0 +1,38 @@
+<div class="buttons">
+	<button data-editor-name="inline" style="font-size:20px">Focus inline</button>
+	<button data-editor-name="classic" style="font-size:20px">Focus classic</button>
+</div>
+
+<div style="height:8000px"></div>
+
+<div id="inline" contenteditable="true">
+	<figure class="easyimage">
```

Comment:
Using widget as the only content inside the editor makes testing if editor is focused more difficult. The most obvious way (start typing) is not working anymore.